zerver/webhooks/gocd/doc.md

```diff
@@ -0,0 +1,21 @@
+Zulip supports integration with GoCD and can notify you of
+your build statuses.
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+1. Add the following to your `Config.XML` file and Push this change to your repository.
+
+    ```
+    <pipeline name="mypipeline">
+        <trackingtool link="http://mingle.thoughtworks.com/go/${ID}" regex="##(\d+)"/>
+        ...
+    </pipeline>
+    ```
+
+    For further information, see [GoCD's documentation](https://docs.gocd.org/current/integration/).
```

Comment:
`Push this change to your repository` should go after this sentence.

Also, where does the webhook URL go? It has to go somewhere for the webhook to be configured, right? If the `mingle.thoughtworks.com` stuff is where it is supposed to go, we should replace that with `<URL constructed above>`. :)